mend_sca_cleanup_tool/sca_cleanup_tool.py

```diff
@@ -368,22 +416,53 @@ def parse_config_file(filepath):
                     project_parallelism_level=config['DEFAULT'].get('ProjectParallelismLevel', 5),
                     dry_run=config['DEFAULT'].getboolean("DryRun", False),
                     skip_report_generation=config['DEFAULT'].getboolean("SkipReportGeneration", False),
-                    skip_project_deletion=config['DEFAULT'].getboolean("SkipProjectDeletion", False)
+                    skip_project_deletion=config['DEFAULT'].getboolean("SkipProjectDeletion", False),
+                    proxy=config['DEFAULT'].getboolean("proxy", '')
                 )
     else:
         print(f"No configuration file found at: {filepath}")
         exit()
 
-def post_api_request(request):
+
+def post_api_request(request, report = False):
     try:
-        MAIN_API_CONNECTION.request("POST", '/api/v1.4', request, HEADERS)
+        #  Using Http request, this code here just for testing requests API call
+        #r = call_api(header=HEADERS, data=request, report=report)
+        #return r
+        MAIN_API_CONNECTION.request("POST", f'/api/v1.4', request, HEADERS)
         return MAIN_API_CONNECTION.getresponse().read()
-    except:
-        sys.exit(f"There was an issue calling the Mend API with URL: {CONFIG.mend_url}")
+    except Exception as err:
+        print(f"Host: {MAIN_API_CONNECTION.host}")
+        r = call_api(header=HEADERS, data=request, report=report)
+        return r
+
+
+def call_api(header, data, method="POST", report = False):
```

Comment:
Method name is too similar to post_api_request, I would make it clear what this api is doing differently from the main call.

Reply:
It is possible, but call_api is more-less universal func that is used in a few of our tools.

Reply:
Right, but in this case it is confusing and not descriptive with the rest of the code. If I was a new programmer picking this up, I wouldn't know which method to call.